src/measure_theory/measurable_space.lean

```diff
@@ -134,15 +135,67 @@ lemma monotone_comap : monotone (measurable_space.comap g) := assume a b h, coma
 lemma comap_map_le : (m.map f).comap f ≤ m := (gc_comap_map f).l_u_le _
 lemma le_map_comap : m ≤ (m.comap g).map g := (gc_comap_map g).le_u_l _
 
+lemma comap_of_involutive {m' : measurable_space β} {g : β → β} (hg : involutive g)
```

Comment:
This lemma immediately follows from 3 facts, 2 of them are not in `mathlib` yet. Could you add them, please?

- [ ] `measurable_embedding.comap_eq`: `comap` of the `measurable_space` instance along a `measurable_embedding` is the corresponding instance;
- [ ] `measurable_equiv.of_involutive`: a measurable involutive map generates a `measurable_equiv`; the `to_equiv` projection is given by `equiv.perm.of_involutive`;
- [X] `measurable_space.comap_comp` - already in the library.

While you're at it, could you please add
- [ ] `measurable_equiv.map_eq`
as well?

I'm not sure if we should use `measurable_space` in the names to make it clear that the lemmas are not about `measure_theory.measure.map` and `measure_theory.measure.comap`.

Reply:
I've added everything you mentioned here, but I couldn't prove `measurable_embedding f ↔ comap f _ = _ ∧ measurable_set (range f)`. Instead I proved `measurable_embedding f ↔ injective f ∧ comap f _ = _ ∧ measurable_set (range f)`.